plugin/trino-hive/src/test/java/io/trino/plugin/hive/metastore/glue/TestHiveGlueMetastore.java

```diff
@@ -207,6 +219,37 @@ protected HiveMetastore createMetastore(File tempDir)
                                 .setHideDeltaLakeTables(true)).get());
     }
 
+    @Test
+    public void cleanupOrphanedDatabases()
```

Comment:
Runs in parallel. Any databases being listed in this test class can then become flaky.

Can we make this just a method and call in @BeforeClass?

Reply:
There is time threshold, so it doesn't matter whether this is `@Test` or `@BeforeClass`.

It still runs in parallel with other runs (test processes), that's why i tried to code this defensively.